README.md

```diff
@@ -1,15 +1,21 @@
 # landing-frontend
-coala Landing Page - https://gitlab.com/coala/landing is needed as backend
-
+This is the coala frontend Landing Page which uses - https://gitlab.com/coala/landing as its backend.
```

Comment:
This repository is the https://coala.io/ website frontend.
It uses https://gitlab.com/coala/landing as its backend.